### PR TITLE
Adapt to Boston

### PR DESCRIPTION
**Pull request by nedbat**
Adapt the existing PyCon language to Boston Python.

I intend to also make more changes, but this is just to get the existing PyCon text into a form that is right for Boston Python.

### CODE REVIEW

**Comment by bhrutledge** [comment]
Thanks for setting this up, @nedbat. Initial scan looks good; I'm planning on doing a deeper read at project night, if not sooner.

File: code_of_conduct.md [hunk 1]
```diff
@@ -34,67 +34,47 @@ Examples of unacceptable behavior by participants include:
...
 
 Attendees are further expected to comply with all state and local laws on this matter.
 
 ## Scope
 
-All PyCon US attendees are subject to the Code of Conduct. This includes PyCon US staff and volunteers, as well as attendees of the tutorials, workshops, summits, poster sessions, receptions, conference, sprints, and sponsored or unofficial events.
+All Boston Python attendees are subject to the Code of Conduct. This includes staff and volunteers, as well as attendees of events, presentations, project nights, after-event gatherings, receptions, and all sponsored or unofficial events.
 
-Exhibitors in the expo hall, sponsor or vendor booths, or similar activities are also subject to the Code of Conduct. In particular, exhibitors should not use sexualized images, activities, or other material. Booth staff (including volunteers) should not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
+Communication outside of events are also subject to the Code of Conduct, including email, online postings, chat messages, and any other communication under the auspices of Boston Python.
```
Comment (bhrutledge) [inline code_of_conduct.md:53]:
```suggestion
Communication outside of events is also subject to the Code of Conduct, including email, online postings, chat messages, and any other communication under the auspices of Boston Python.
```

**Review by bhrutledge — APPROVED** [review]
@nedbat Looks like a good place to start, with one grammar tweak.

In the email thread, you mentioned liking the language of [PyCascades](https://2019.pycascades.com/code-of-conduct/). That's ultimately based on [Django's](https://www.djangoproject.com/conduct/), which seems a little smaller in scope, and might be easier to adapt or borrow from. Another related one is [WriteTheDocs](https://www.writethedocs.org/code-of-conduct/), which has a nice [short form](http://www.writethedocs.org/code-of-conduct-shortform-meetups/).

**Comment by nedbat** [comment]
I'm merging this while I mull over the Write The Docs code, which I like a lot.

**Pull request merged**